synapse/federation/transport/server.py

```diff
@@ -288,20 +293,27 @@ def _wrap(self, func):
                 logger.warn("authenticate_request failed: %s", e)
                 raise
 
-            # Start an opentracing span
-            with start_active_span_from_context(
-                request.requestHeaders,
-                "incoming-federation-request",
-                tags={
-                    "request_id": request.get_request_id(),
-                    tags.SPAN_KIND: tags.SPAN_KIND_RPC_SERVER,
-                    tags.HTTP_METHOD: request.get_method(),
-                    tags.HTTP_URL: request.get_redacted_uri(),
-                    tags.PEER_HOST_IPV6: request.getClientIP(),
-                    "authenticated_entity": origin,
-                    "servlet_name": request.request_metrics.name,
-                },
-            ):
+            tags = get_tags()
+            _tags = {
```

Comment:
I tend to avoid using variables that start with `_`, as in my mind they either mean "unused" or "private instance variables". Maybe something descriptive like `request_tags` would work.